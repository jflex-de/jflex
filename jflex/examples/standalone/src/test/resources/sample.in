name someone

Hello
This is a sample input file for the
standalone example scanner.

Have a nice day!

