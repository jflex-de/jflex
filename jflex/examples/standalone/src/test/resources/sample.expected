

Hello someone!
This is a sample input file for the
standalone example scanner.

Have a nice day!

